examples/xoxo/xoxo.py

```diff
@@ -0,0 +1,183 @@
+from pysmt.shortcuts import FreshSymbol, Symbol, String, LE, GE, Int, Not, Or, And, Equals, Plus, Solver, StrContains
+from pysmt.typing import INT, STRING
```

Comment:
It does not seem like you are using the theory of STRING. Consider cleaning-up these unused imports.

Reply:
done